src/google/protobuf/descriptor.cc

```diff
@@ -7683,7 +7683,7 @@ bool DescriptorBuilder::OptionInterpreter::SetOptionValue(
       if (uninterpreted_option_->has_positive_int_value()) {
         if (uninterpreted_option_->positive_int_value() >
             static_cast<uint64_t>(std::numeric_limits<int32_t>::max())) {
-          return AddValueError("Value out of range for int32 option \"" +
+          return AddValueError("Value out of range, -2,147,483,648 to 2,147,483,647, for int32 option \"" +
```

Comment:
I was a bit worried that parens would make it look like a closed range (e.g. `(` vs `[` for range notation). Building on the other comment, to use formatted numeric constants instead of string literals for these, it looks better when the numbers are formatted without commas. Take a look and latest commit and let me know what you think.